src/get_property_names_args_builder.rs

```diff
@@ -67,12 +78,7 @@ impl GetPropertyNamesArgsBuilder {
   }
 
   pub fn build(&self) -> GetPropertyNamesArgs {
-    GetPropertyNamesArgs {
-      mode: self.mode,
-      property_filter: self.property_filter,
-      index_filter: self.index_filter,
-      key_conversion: self.key_conversion,
-    }
+    Default::default()
```

Comment:
I think this was correct before

Reply:
@bartlomieju yes, you're right! Not sure how to get rid of the duplication though